src/UhdmAst.cpp

```diff
@@ -441,23 +441,32 @@ AstNodeDType* getDType(vpiHandle obj_h, UhdmShared& shared) {
         // todo: static/dynamic/assoc/queue
         auto rand_type = vpi_get(vpiRandType, obj_h);
         // todo: rand/randc/notrand
-        AstNodeDType* element_dtype = nullptr;
+        AstNodeDType* element_dtypep = nullptr;
```

Comment:
Use `camelCase` instead of `snake_case` for non-VPI-related variables. (Yes, I know, this file is not consistent with that)

Reply:
Done. Thank you